src/libs/API/types.ts

```diff
@@ -526,6 +527,7 @@ type ReadCommandParameters = {
     [READ_COMMANDS.OPEN_POLICY_MORE_FEATURES_PAGE]: Parameters.OpenPolicyMoreFeaturesPageParams;
     [READ_COMMANDS.OPEN_POLICY_ACCOUNTING_PAGE]: Parameters.OpenPolicyAccountingPageParams;
     [READ_COMMANDS.SEARCH]: Parameters.SearchParams;
+    [READ_COMMANDS.OPEN_SUBSCRIPTION_PAGE]: EmptyObject;
```

Comment:
```suggestion
    [READ_COMMANDS.OPEN_SUBSCRIPTION_PAGE]: null;
```

Let's avoid usage of `EmptyObject` here as we are removing them in another PR

Reply:
All right, thanks!